pages/apim/3.x/installation-guide/configuration/repositories/installation-guide-repositories-jdbc.adoc

```diff
@@ -34,16 +34,20 @@ This repository plugin is for connecting to SQL databases.
 |https://docs.microsoft.com/en-us/sql/connect/jdbc/download-microsoft-jdbc-driver-for-sql-server?view=sql-server-2017[Download page]
 |===
 
-== Install the JDBC plugin
+== Install
 NOTE: You need to repeat the following steps on each component (APIM Gateway and APIM API) where the SQL database is used.
 
+== Install the JDBC plugin (before 3.5)
+NOTE: Because the plugin is part of the default distribution since version 3.5 you may skip this step than
```

Comment:
NOTE: Because the plugin is part of the default distribution from APIM version 3.5, you only need to complete these steps for version 3.4 or earlier.

Repeat the following steps on each component (APIM Gateway and APIM API) where the SQL database is used.